### PR TITLE
Bogdannegura welcome

### PR DESCRIPTION
**Pull request by BogdanNegura**

### CODE REVIEW

File: src/_component/welcome/welcome.pug [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.welcome
+    .red-box
+        h3.section-title Welcome to My CV
+    h1.my-name Bogdan Negura
+    h2.job-position Junior Front-End Developer
+    h4.subtitle  About me
```
Comment (marianzburlea) [inline src/_component/welcome/welcome.pug]: broken logic, h1, h2, h3, h4, not h3, h1, h2, h4

File: src/_component/welcome/welcome.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.text-decription {
+  background: white;
+  padding: 20px;
+  line-height: 2;
+  font-size: 14px;
+  text-align: left;
+  border-radius: 6px;
+  box-shadow: 0 0 5px #707070, 0 0 5px #e6e0e0;
```
Comment (marianzburlea) [inline src/_component/welcome/welcome.scss]: nice one, double shadow
💯

File: src/index.pug [hunk 3]
```diff
@@ -0,0 +1,18 @@
+doctype html
+html
+    head
+        meta(name="viewport" content="width=device-width")
+        meta(name="description" content="This is the version 2 of My-cv for junior and we gonna use pug as templete engine for hrml")
```
Comment (marianzburlea) [inline src/index.pug]: use multiline for tag

File: src/index.pug [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        title Welcome to My CV Version 2 Junior
+        //- link(rel="stylesheet", href="./style.scss")
+        if config.entry.css.embed
```
Comment (marianzburlea) [inline src/index.pug]: if and the line after it are at the same level. there is no purpose for this.
indent the next one so it gets affected by the if condition

File: src/index.pug [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        link(rel="stylesheet" href=`${embedPath}` embed)
+    body    
+        .content
+            include _component/Welcome/Welcome
```
Comment (marianzburlea) [inline src/index.pug]: WHY OH WHY do you use capital letters in naming folders and files?
don't become @CarolEnache

Comment (BogdanNegura) [reply src/index.pug]: image
I'm using normal writing, probably the autogenerate for the path does it.

Comment (marianzburlea) [reply src/index.pug]: well, no more not paying attention then, we're coders here, not casual text chatters

File: src/_component/layout/font.scss [hunk 6]
```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "icon";
+  src: url("asset/font/icon-v1.0/icon.ttf?wznzm1") format("truetype"),
+    url("asset/font/icon-v1.0/icon.woff?wznzm1") format("woff"),
+    url("asset/font/icon-v1.0/icon.svg?wznzm1#icon") format("svg");
```
Comment (marianzburlea) [inline src/_component/layout/font.scss]: You can now use *baseUrl* in scss.

**Pull request merged**